<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDialog</class>
 <widget class="QDialog" name="ChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>432</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>221</width>
    <height>432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/art/d.png);</string>
   </property>
   <property name="text">
    <string>是社长找我啊！好像加社这么久也没有和社内进行交流像透明人……</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>221</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/weixin2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>点击查看新消息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
